__tests__/app.test.js

```diff
@@ -270,28 +272,256 @@ describe("POST /api/articles/:article_id/comments", () => {
     expect(response.body.msg).toBe(expected);
   });
 
-  test('400: Should return 400 if the body is missing from body request', async () => {
-    const response = await request(app)
-      .post('/api/articles/9/comments')
-      .send();
-    
+  test("400: Should return 400 if the body is missing from body request", async () => {
+    const response = await request(app).post("/api/articles/9/comments").send();
+
     expect(response.status).toBe(400);
-    expect(response.body.msg).toBe('Request body is missing');
+    expect(response.body.msg).toBe("Request body is missing");
   });
-  test('400: Should return 400 if the author is missing from body request', async () => {
+  test("400: Should return 400 if the author is missing from body request", async () => {
     const response = await request(app)
-      .post('/api/articles/9/comments')
-      .send([{ body: 'Amazing!' }]); 
-    
+      .post("/api/articles/9/comments")
+      .send([{ body: "Amazing!" }]);
+
     expect(response.status).toBe(400);
-    expect(response.body.msg).toBe('Author is required');
+    expect(response.body.msg).toBe("Author is required");
   });
-  test('400: Should return 400 if the body is missing from body request', async () => {
+  test("400: Should return 400 if the body is missing from body request", async () => {
     const response = await request(app)
-      .post('/api/articles/9/comments')
-      .send([{ author: 'butter_bridge' }]); 
-    
+      .post("/api/articles/9/comments")
+      .send([{ author: "butter_bridge" }]);
+
     expect(response.status).toBe(400);
-    expect(response.body.msg).toBe('body text is required');
+    expect(response.body.msg).toBe("body text is required");
+  });
+});
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+describe("GET /api/articles/:article_id (comment_count)", () => {
+  test("Should return an array of the object article requested by id with the addition of comment_count", async () => {
```

Comment:
This endpoint should respond with a single article object rather than an array. There's only ever one article with that id so the array isn't needed